index.html

```diff
@@ -117,31 +148,35 @@ <h3>TRAVEL STORIES</h3>
                                 <img class="mobile-story-image" src="./images/Rectangle-15.jpeg" alt="story1">
                                 <div class="story-text">
                                     <h4>10 Photos of Attractive Thailand</h4>
-                                    <p class="story-content">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.Lorem ipsum dolor sit amet, consectetur adipiscing elit...<a href="#">Read More</a></p>
+                                    <p class="story-content-desktop">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.Lorem ipsum dolor sit amet, consectetur adipiscing elit...<a href="#">Read More</a></p>
```

Comment:
@PavelTsikhanau Here and elsewhere: Why duplicate the same content and layout for mobile?

Reply:
@masontikhonov 
In the mobile version the text is a couple of sentences shorter

Reply:
Ah apologies, make sense